src/components/user_view.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import { fetchUsersServices, fetchServiceAndOwner } from '../actions/index';
```

Comment:
you shouldnt need fetchserviceandowner here i dont think?

Reply:
Well fetchUsersServices was returning everything except the comments and ratings. And I needed all of the comments and ratings, so that's why I was using fetchServiceAndOwner for each service lol. But, it got really messy. I can take it out

Reply:
tooketh

Reply:
ahh okay. maybe submit a backend issue. or also just create a new action creator for just fetching a service.